examples/fpga/artya7/top_artya7.core

```diff
@@ -28,7 +33,7 @@ parameters:
   SRAMInitFile:
     datatype: str
     description: SRAM initialization file in vmem hex format
-    default: "../../../../../examples/sw/led/led.vmem"
+    default: "../../../../../examples/sw/benchmarks/coremark/coremark.vmem"
```

Comment:
Do you really want to change the default? If yes, you should also adapt the `README.md` accordingly: https://github.com/lowRISC/ibex/blob/master/examples/fpga/artya7/README.md (it still mentions `led.vmem`)

Reply:
I think it is better to specify it when calling fusesoc. Where should I explain how to use the power analysis option?

Reply:
I think you could add a new section to the README.md under https://github.com/lowRISC/ibex/blob/master/examples/fpga/artya7/README.md